presto-docs/src/main/sphinx/release/release-330.rst

```diff
@@ -0,0 +1,131 @@
+* Add ``physical_input_bytes`` column to ``system.runtime.tasks`` table. (:issue:`2803`)
+* Verify that the target schema exists for the :doc:`/sql/use` statement. (:issue:`2764`)
+* Verify that the session catalog exists when executing :doc:`/sql/set-role`. (:issue:`2768`)
+
+
+Server Changes
+--------------
+
+* Require running on Java 11. This requirement may be temporarily relaxed by adding
+  ``-Dpresto-temporarily-allow-java8=true`` to the Presto :ref:`presto_jvm_config`.
+  This fallback will be removed in future versions of Presto after March 2020. (:issue:`2751`)
+* Add experimental support for running on Linux aarch64 (ARM64). (:issue:`2809`)
+
+Security Changes
+----------------
+
+* :ref:`principal-rules` are deprecated and will be removed in a future release.
+  These rules have been replaced with :doc:`/security/user-mapping`, which
+  specifies how a complex authentication user name is mapped to a simple
+  user name for Presto, and :ref:`impersonation-rules` which control the ability
+  of a user to impersonate another user. (:issue:`2215`)
+* A shared secret is now required when using :doc:`security/internal-communication`.  (:issue:`2202`)
+* Kerberos for :doc:`security/internal-communication` has been replaced with the new shared secret mechanism.
+  The ``internal-communication.kerberos.enabled`` and ``internal-communication.kerberos.use-canonical-hostname``
+  configuration properties must be removed. (:issue:`2202`)
+* When authentication is disabled, the Presto user may now be set using standard
+  HTTP basic authentication with an empty password.  (:issue:`2653`)
+
+Web UI Changes
+--------------
+
+* Display physical read time in detailed query view. (:issue:`2805`)
+
+JDBC Driver Changes
+-------------------
+
+* Fix a performance issue on JDK 11+ when connecting using HTTP/2. (:issue:`2633`)
+* Implement ``PreparedStatement.setTimestamp`` variant that takes a ``Calendar``. (:issue:`2732`)
+* Add ``roles`` property that allows to configure authorization roles to be used for catalogs. (:issue:`2780`)
+* Add ``sessionProperties`` property that allows to configure system and catalog session properties. (:issue:`2780`)
+* Allow passing ``:`` character within a value of extra credential passed with ``extraCredentials``. (:issue:`2780`)
+
+CLI Changes
+-----------
+
+* Fix a performance issue on JDK 11+ when connecting using HTTP/2. (:issue:`2633`)
+
+Cassandra Connector Changes
+---------------------------
+
+* Fix query failure when identifiers should be quoted. (:issue:`2455`)
+
+Hive Connector Changes
+----------------------
+
+* Fix reading symlinks from kerberized HDFS. (:issue:`2720`)
+* Allow using writer scaling with all file formats. Previously, it was not supported for
+  text-based, SequenceFile, or Avro formats. (:issue:`2657`)
+* Add support for symlink-based tables with Avro files. (:issue:`2720`)
+* Allow to ignore partitions that do not have matching data directory. This can be controlled with
```

Comment:
`Add support for ignoring partitions with a non-existent data directory.`?

Reply:
```
This can be configured using the ``hive.ignore-absent-partitions=true`` configuration property
or the ``ignore_absent_partitions`` session property.